app/Models/JobPoster.php

```diff
@@ -589,4 +589,15 @@ public function getClassificationMessageAttribute()
         }
         return null;
     }
+
+    /**
+     *
+     * Job Poster is from the Emergency Response department
+     *
+     * @return boolean
+     */
+    public function isInEmergencyResponseDepartment()
+    {
+        return $this->department->name == 'Emergency Response';
```

Comment:
Careful here. There's a few issues with checking the name. First, if the request was in french `->name` will be different. Also,  admins could change the name on the Admin portal without telling us.

Checking against any hardcoded value is way less than ideal, but lets use the id instead, as its less likely to change. The id of this department on production is **18**.

Reply:
Ya I wasn't sure about this yet. Thanks.